homeassistant/components/google/quality_scale.yaml

```diff
@@ -0,0 +1,76 @@
+rules:
+  # Bronze
+  config-flow:
+    status: done
+    comment: Integration uses an OAuth config flow.
```

Comment:
data_description is missing in the options flow

Reply:
Marked as todo.